tests/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 FetchContent_Declare(
   doctest
   GIT_REPOSITORY "https://github.com/onqtam/doctest"
```

Comment:
This just upgrades to the latest version?

Reply:
Yea sorry, I kind of snuck that in. I was hoping it would resolve a cmake warning but it seems thats not in the latest doctest release yet.